lib/ex_plasma/utxo.ex

```diff
@@ -40,23 +40,22 @@ defmodule ExPlasma.Utxo do
           owner: Transaction.address() | Transaction.address_hash()
         }
 
-  @empty_integer 0
-  @empty_address to_hex(<<0::160>>)
-
   # Currently this is the only output type available.
   @payment_output_type 1
 
   # Contract settings
   @block_offset 1_000_000_000
   @transaction_offset 10_000
+  @max_txindex :math.pow(2, 16) - 1
+  @max_blknum (:math.pow(2, 54) - 1 - @max_txindex) / (@block_offset / @transaction_offset)
```

Comment:
also... if there's a way (I'm sure it has to be) for us to retrieve this value from the contracts? this should be ex_plasma mandatory configuration on boot. otherwise we're going to have similar problems as in elixir-omg with free floating constants where we hope they match with contracts.

Reply:
Yep, will add a comment. Also opened an issue here for us to move towards getting this from the contract directly: https://github.com/omisego/ex_plasma/issues/26